conf/github-contributions.nginx.conf

```diff
@@ -31,6 +31,11 @@ server {
         autoindex on;
     }
 
+    location /sitemap {
+        alias /srv/sitemap/;
+        autoindex on;
+    }
```

Comment:
There's going to be `/sitemap.xml`, then that ends up pointing to a lot of different files named `/sitemap/sitemap-[aa,ab,etc].txt` because of limits on sitemap.xml/sitemap.txt's size.